edpm_ansible/ansible_plugins/modules/container_config_hash.py

```diff
@@ -145,6 +139,42 @@ def _update_container_config(self, path, config):
         os.chmod(path, 0o600)
         self.results['changed'] = True
 
+    def _tar_cleanup(self, tarinfo):
+        tarinfo.mtime = 0
+        return tarinfo
+
+    def _create_checksum(self, config_volume):
+        """Create an hash from a tar-ed version of the given
+        folder
+
+        :param config_volume: string
+        :returns: string
+        """
+
+        total_hash = hashlib.new('md5')
+        tmp = tempfile.TemporaryFile()
```

Comment:
I think we no longer need tar file as it was used to just join file contents, which can be implemented differently here.

Reply:
100% agree, see new commits.